pkgs/servers/home-assistant/component-packages.nix

```diff
@@ -897,7 +897,7 @@
     "vera" = ps: with ps; [ ]; # missing inputs: pyvera
     "verisure" = ps: with ps; [ jsonpath ]; # missing inputs: vsure
     "versasense" = ps: with ps; [ ]; # missing inputs: pyversasense
-    "version" = ps: with ps; [ pyhaversion ];
+    "version" = ps: with ps; [ ]; # missing inputs: pyhaversion
```

Comment:
This is somehow due to 0b371941a1d8044fa09919a30daaeb00c55a8f16. @fabaff

Reply:
Yes, it looks that way.

Reply:
You disabled the package for python >= 3.8.